stable/external-dns/Chart.yaml

```diff
@@ -1,6 +1,6 @@
 apiVersion: v1
 name: external-dns
-version: 2.16.3
+version: 2.16.4
```

Comment:
As this is a new feature and not bug fixing, you should increment the chart version in the `Chart.yml` according to [semver](http://semver.org/), in this case `MINOR`:

> Given a version number `MAJOR.MINOR.PATCH`, increment the:
> - MAJOR version when you make incompatible API changes,
> - MINOR version when you add functionality in a backwards-compatible manner, and
> - PATCH version when you make backwards-compatible bug fixes.

Reply:
done